internal/backends/sqlite/collection.go

```diff
@@ -64,9 +66,43 @@ func (c *collection) Query(ctx context.Context, params *backends.QueryParams) (*
 		}, nil
 	}
 
-	q := fmt.Sprintf(`SELECT %s FROM %q`, metadata.DefaultColumn, meta.TableName)
+	var filter string
+	var args []any
 
-	rows, err := db.QueryContext(ctx, q)
+	iter := params.Filter.Iterator()
```

Comment:
There is no need to iterate. The only currently supported filter should have exactly one field: `{_id: ObjectId("123")}`